test/new-e2e/tests/installer/all_packages_test.go

```diff
@@ -92,6 +92,11 @@ func TestPackages(t *testing.T) {
 				if flavor.Flavor == e2eos.Fedora {
 					flake.Mark(t)
 				}
+				if strings.Contains(suite.Name(), "upgrade") {
+					// TODO(baptiste): Remove the flake once 0.4.7 is released
+					// it is currently broken as we are changing the socket path
+					flake.Mark(t)
```

Comment:
shouldn't this still work?
also to avoid breaking the injector and be safe in transitions I think we should parse both files for couple versions (1 month+)

Reply:
With the global symlink it should yes, I'll wait for a build to test this locally.

> also to avoid breaking the injector and be safe in transitions I think we should parse both files for couple versions (1 month+)

You mean the injector right?

Reply:
Removed, it works with the global symlink